doc/abrt-xorg.conf.txt

```diff
@@ -11,16 +11,23 @@ SYNOPSIS
 
 DESCRIPTION
 -----------
-The 'abrt-action-analyze-xorg' tool processes Xorg crash problem data
-and generates deduplication hashes for it.
+The 'abrt-dump-journal-xorg' tool extracts Xorg crashes from systemd-journal and
+creates the associated problem directories. The 'abrt-action-analyze-xorg' tool
```

Comment:
... creates problem directories for them ... ?

Does it even makes sence to have it in documentation about conf file?

Reply:
It doesn't, most likely. I'll remove this paragraph.